CHANGELOG.md

```diff
@@ -4,6 +4,28 @@
 
 <!-- changelog start -->
 
+## [0.15.0 (2024-06-06)]
```

Comment:
```suggestion
## [0.16.0 (2024-06-06)]
```

Reply:
🤔  was this auto generated ?

Reply:
Good catch, does it read the version from the Cargo.toml at the time the script is ran?

Reply:
ah it does yeah. I think this implies the cargo.toml was bumped after the changelog generation ?

Reply:
Correct, at some point the publish script can take care of this to avoid manual errors.